playlist_test.go

```diff
@@ -536,6 +536,35 @@ func TestReplacePlaylistTracks(t *testing.T) {
 	}
 }
 
+func TestReplacePlaylistItems(t *testing.T) {
```

Comment:
It would be nice if this was a table driven test for all scenarios in `ReplacePlaylistItems` rather than two seperate tests :)

Reply:
Replace the 2 separate tests with table tests